src/components/ProjectHeader/ProjectHeader.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { FeaturedImage, Heading } from 'components';
+import styles from './ProjectHeader.module.scss';
+
+function ProjectHeader({ image, alt = '', title, summary }) {
+  return (
+    <section className={styles['header']}>
+      <div className="container">
+        <div className="row row-center">
+          <FeaturedImage
+            className="column column-50"
+            image={image}
+            alt={alt}
```

Comment:
I don't think this prop matters since the alt text is devived from `image?.altText` in `<FeaturedImage>`:

https://github.com/wpengine/atlas-blueprint-portfolio/blob/f48f7eebefd2f927fcfd305be997e55f644190ca/src/components/FeaturedImage/FeaturedImage.js#L6